### DISCUSSION TITLE
DISCUSSION: DataFusion Meetup in San Francisco Bay Area USA

### DISCUSSION THREAD

**Discussion by alamb** [apache/datafusion#10340] [Ideas]
After our first meetup, (see https://github.com/apache/datafusion/discussions/8152 / https://github.com/apache/datafusion/discussions/8522) I have heard there may be interest in other meetups in different geographies. 

I would personally love to support such meetups (will promote, try to attend, can help find sponsors, etc) as much as possible but I don't have the bandwidth to actually organize them.

Anyone who is interested in attending or helping to organize please leave a comment

Other meetup discussions:
https://github.com/apache/datafusion/discussions/10347

**Comment by trungda** [comment]
Interested in attending. My work is not directly relevant (in the OLTP space) and I haven't contributed to datafusion enough so not sure about hosting.

**Comment by alamb** [comment]
Does either the weeks of June 24 or July 1 work for anyone in the bay area? I think @houqp  is based there as is @sunchao

**Reply by andygrove** [reply]
Also, @viirya and some other Comet contributors are there

**Comment by cpard** [comment]
Happy to help with the logistics! Whoever else would like to help with that, just let me know.

**Comment by cpard** [comment]
Adding @changhiskhan to the conversation. It will be awesome to hear the story of how Lance is utilizing DataFusion!

**Comment by andygrove** [comment]
I would be happy to come and talk about Comet. I'll need to get approval and am working on that.

**Comment by alamb** [comment]
I think we are looking at June 24, 2024 as a date

**Comment by emgeee** [comment]
I'm in SF and would love to attend. I can also help organize and possibly provide office space and some sponsorship.

**Reply by cpard** [reply]
Hey @emgeee let's chat about this! I've been working on the logistics of the meetup and it would be awesome to have more people involved!

**Reply by emgeee** [reply]
Great, let me know the best way to connect

**Reply by cpard** [reply]
Just reached out to you on X

**Comment by yyy1000** [comment]
I'm a new contributor to datafusion and will be in bay area this summer. I'd like to attend too.

**Comment by killme2008** [comment]
Interested in attending. We are building GreptimeDB based on datafusion.

**Comment by alamb** [comment]
We have a meetup scheduled Moday June 24. at 5PM  register at https://lu.ma/6bphole2 (see more details https://github.com/apache/datafusion/discussions/10800 / )

**Reply by tisonkun** [reply]
Cool! I'll arrive at Bay Area at June 21st. Hopefully I can apply for a topic here.

**Reply by tisonkun** [reply]
I've registered at the event and I found that there is already a ​speaker lineup. How can I apply for my topic or it's closed?

**Reply by alamb** [reply]
Please email me at alamb@influxdata.com -- we have a space for you.

**Reply by tisonkun** [reply]
@alamb SGTM. Emailed.

**Comment by cpard** [comment]
Hey everyone,

The San Francisco Bay Area meetup is now officially behind us and it was an amazing experience!

A few of the folks who participated sent some feedback on lu.ma and I thought that it will be a much better place to provide feedback here on GH instead.

For the people who attended, please let us know of your comments. 

Let us know of what you think was good and most importantly what could have been better.

Also, please share any ideas for future events. We definitely want to repeat the meetup and I'm sure we can create an even better experience together.

**Comment by emgeee** [comment]
Hey everyone, we're happy to announce the next San Francisco DataFusion meetup has been scheduled for September 25th! If you're interested in attending please sign up here: https://lu.ma/05lyf19h

**Comment by alamb** [comment]
We are plotting another meetup during DataBricks Data and AI event:
- https://github.com/apache/datafusion/discussions/15657

**Comment by alamb** [comment]
Working on next one here https://github.com/apache/datafusion/discussions/25665
